test/seldon/setup/manifests/model/seldon-model-psp-rolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
this should be removed

Reply:
yaml file removed along with the line in script that references it.